src/qml/components/StorageSettings.qml

```diff
@@ -18,14 +18,14 @@ ColumnLayout {
         Layout.fillWidth: true
         header: qsTr("Storage limit")
         actionItem: ValueInput {
-            description: qsTr("2 GB")
+            description: "2 GB"
```

Comment:
Not sure about this.

Reply:
Yeah this should still be translated. The translator should be parameterized in this case to convert the GB but not the number.

Reply:
ok reverted this line, Still not sure how this will actually end up. The component that ends up being the real number input for our options can either split up the value and the units or paramterize the value. We will see what plays best into the validation that we need to do on the input value.

Reply:
Depending on how you implement it. You can use format strings and add args

text: qsTr("File %1 of %2").arg(counter).arg(total)
 https://doc.qt.io/qt-5.15/qtquick-internationalization.html